src/ios/mbprogress-hud.ts

```diff
@@ -48,6 +49,9 @@ export class LoadingIndicator {
       if (ios.backgroundColor) {
         this._hud.color = new Color(ios.backgroundColor).ios;
       }
+      if(!ios.userInteractionEnabled){
```

Comment:
Please change this conditional to:
```
if(ios.userInteractionEnabled === false){
...
```
We don't want to require this be set and be default a `falsey` value (for example, `undefined`) would cause this to be set to false but the default should be `true` when not set at all.